src/orders/utils.ts

```diff
@@ -46,6 +47,7 @@ export const getPostCollectionOfferPayload = (
       collection: { slug: collectionSlug },
     },
     protocol_data,
+    protocol_address: SEAPORT_CONTRACT_VERSION_V1_5,
```

Comment:
instead of adding this here, wondering if you could define a `DEFAULT_SEAPORT_CONTRACT_VERSION = SEAPORT_CONTRACT_VERSION_V1_5` and use that everywhere, then in the future just have to change one line to update version